hdf5.spec

```diff
@@ -14,14 +14,15 @@
 %global snaprel %{nil}
 %global hdf5_major 1
 %global hdf5_minor 13
-%global hdf5_bugfix 0
-%global hdf5_prerelease rc5
-%global hdf5_tag %{hdf5_major}_%{hdf5_minor}_%{hdf5_bugfix}%{?hdf5_prerelease:-%{hdf5_prerelease}}
+%global hdf5_bugfix 1
+#global hdf5_prerelease rc5
+%global hdf5_tag %{hdf5_major}_%{hdf5_minor}_%{hdf5_bugfix}%{?hdf5_prerelease:~%{hdf5_prerelease}}
```

Comment:
We don't want to change the pre-release separator here from `-` to `~` as this is the macro used to generate the name of the upstream tarball to download and as of their most recent pre-release, it seems they are still using `-` and not `~`.

It is still correct though that we change the `-` pre-release separator to `~` in the `Version:` tag on line 24.

Reply:
i see. i must have copied this from there then instead of reverting the change. should be fixed now.